src/sentry/models/monitor.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import absolute_import, print_function
+
+from croniter import croniter
+from datetime import datetime, timedelta
+from django.db import models
+from django.utils import timezone
+from uuid import uuid4
+
+from sentry.constants import ObjectStatus
+from sentry.db.models import (
+    GUIDField,
+    Model,
+    BoundedPositiveIntegerField,
+    EncryptedJsonField,
+    sane_repr,
+)
+
+
+def generate_secret():
+    return uuid4().hex + uuid4().hex
```

Comment:
You can get this function from `sentry.models.apitoken:generate_token`

Reply:
the way django 1.8+ generates migration the default has to be module-scoped, which is the only reason im not using a lambda here. I generally frown upon using locally scoped functions across multiple modules (e.g. APIToken has nothing to do with Monitor), and given that its for a specific field default I think its fine to duplicate the code